notifications_utils/template.py

```diff
@@ -164,7 +164,12 @@ def __init__(
     @property
     def content_with_placeholders_filled_in(self):
         # We always call SMSMessageTemplate.__str__ regardless of
-        # subclass, to avoid any HTML formatting
+        # subclass, to avoid any HTML formatting. SMS templates differ
+        # in that the content can include the service name as a prefix.
+        # So historically we’ve returned the fully-formatted message,
```

Comment:
To make sure I'm clear, when we say fully-formatted message is that 'service name: content' vs just 'content'? And you say 'plain text representation of the content'. Is the word 'plain-text' important, in that the fully-formatted message isn't plaintext?

Reply:
It’s also about other normalisation, for example removing multiple newlines and spaces before punctuation.

For emails and letters the fully-formatted message is HTML, as opposed to plain text.

Reply:
It’s also about other normalisation, for example removing multiple newlines and spaces before punctuation.

For emails and letters the fully-formatted message is HTML, as opposed to plain text.